src/main/resources/application.yml

```diff
@@ -6,6 +6,8 @@ hostiflix-login-redirect:
 
 deploymentServiceUrl: http://deployment-service/jobs/submit
```

Comment:
I'd also change this to `deployment-service-url` to match our naming scheme with the dashes.